jmx-metrics/build.gradle.kts

```diff
@@ -13,6 +13,7 @@ application.mainClass.set("io.opentelemetry.contrib.jmxmetrics.JmxMetrics")
 
 repositories {
   mavenCentral()
+  mavenLocal()
```

Comment:
Is it intentional change? Do we have to modify jmx-metrics?

Reply:
We talked about this in the SIG -- it's was a stop-gap measure in order to consume fresh builds of the agent artifacts...but do we still need it here? I thought we agreed that snapshots would work.

Reply:
Good point, it's a leftover from using a local snapshot.